src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,32 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
+    .findAll({
+      include: [
+        {
+          model: Hacker,
+          required: true,
+        },
+      ],
+    })
+    .then(applications => Promise.all(applications.map(appl => appl.hacker.getResponseStatus(appl).then(responseStatus => ({
```

Comment:
For the record, I have tested with the full set and it does take a good few seconds to complete